Building of vmaf not working

```
12:24:09   Running git clone for vmaf...
12:24:29 ┌ vmaf git  ................................. [Recently updated]
12:24:29 ├ Running clean...
12:24:29 ├ Running make...
Likely error (tail of the failed operation logfile):
ninja: Entering directory `.'
[1/2] g++ @tools/vmafossexec.exe.rsp
FAILED: tools/vmafossexec.exe
g++ @tools/vmafossexec.exe.rsp
C:/Development/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/Development/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/libgcc_eh.a(unwind-seh.o):(.text+0x3d0): multiple definition of `_Unwind_Resume'; src/libvmaf.dll.a(d000018.o):(.text+0x0): first defined here
collect2.exe: error: ld returned 1 exit status
[2/2] "C:/Development/MABS/msys64/mingw64/bin/meson" "--internal" "vcstagger" "../include/vcs_version.h.in" "include/vcs_version.h" "1.3.16" "C:/Development/MABS/build/vmaf-git/libvmaf/include" "@VCS_TAG@" "(.*)" "C:/Development/MABS/msys64/usr/bin/git.EXE" "--git-dir" "C:/Development/MABS/build/vmaf-git/libvmaf/../.git" "describe" "--tags" "--long" "--match" "?.*.*" "--always"
fatal: Not a valid object name .ninja_log
ninja: build stopped: subcommand failed.
make: *** [Makefile:3: all] Error 1
make failed. Check C:/Development/MABS/build/vmaf-git/ab-suite.make.log
This is required for other packages, so this script will exit.
12:24:38   Creating diagnostics file...

All relevant logs have been anonymously uploaded to https://0x0.st/zg03.zip
Copy and paste [logs.zip](https://0x0.st/zg03.zip) in the GitHub issue.
Make sure the suite is up-to-date before reporting an issue. It might've been fixed already.
Try running the build again at a later time.
```

--- a/build/media-suite_compile.sh
+++ b/build/media-suite_compile.sh
@@ -1596,15 +1596,14 @@ if enabled libxvid && [[ $standalone = y ]] && ! { files_exist "${_check[@]}" &&
     do_checkIfExist
 fi
 
-_check=(libvmaf.{a,h,pc})
+_check=(libvmaf.{a,pc} libvmaf/libvmaf.h)
 if [[ $bits = 32bit ]]; then
     do_removeOption --enable-libvmaf
 elif [[ $ffmpeg != "no" ]] && enabled libvmaf &&
     do_vcs "https://github.com/Netflix/vmaf.git"; then
     do_uninstall share/model "${_check[@]}"
-    log clean make clean
-    do_make INSTALL_PREFIX="$LOCALDESTDIR"
-    do_makeinstall INSTALL_PREFIX="$LOCALDESTDIR"
+    do_patch "https://github.com/Netflix/vmaf/compare/master...1480c1:cmake.patch" am
+    do_cmakeinstall
     do_checkIfExist
 fi
 

